packages/features/ee/workflows/lib/alphanumericSenderIdSupport.ts

```diff
@@ -0,0 +1,87 @@
+import { SENDER_ID } from "@calcom/lib/constants";
+
+export function getSenderId(phoneNumber?: string | null, sender?: string | null) {
```

Comment:
@emrysal added the `getSenderId` function and also added a new constant `SENDER_ID` which is the new `NEXT_PUBLIC_SENDER_ID` env variable (if empty, "Cal" as a fallback)